Data/Sys/GameSettings/SRO.ini

```diff
@@ -0,0 +1,18 @@
+# SROENS - Rodea The Sky Soldier
+[Core]
```

Comment:
Small nitpick: There should be a blank line before [Core]. Also, you can write `SROENS, SROJQC, SROPNS` instead of just `SROENS` since all of those IDs are known.